Cargo.toml

```
@@ -14,6 +14,7 @@ readme = "README.md"
 repository = "https://github.com/tauri-apps/tao"
 documentation = "https://docs.rs/tao"
 categories = [ "gui" ]
+exclude = ["audits/**"]
```

we'll still end up with a lot of unnecessary stuff (though small in size), what about
```suggestion
include = ["/README.md", "src/**/*.rs", "LICENSE*"]
```
(cargo.toml/.lock are included by cargo anyway)

---

Does exclude examples impact `cargo doc` on crates.io? If not then we could go with this

---

ohh good question 🤔

---

i added it to the suggestion just in case